src/network/peers.rs

```diff
@@ -273,15 +273,38 @@ impl<N: Network, E: Environment> Peers<N, E> {
                 }
                 // Spawn a handler to be run asynchronously.
                 else {
-                    // Ensure the connecting peer has not surpassed the connection frequency limit.
-                    let last_seen = self.seen_inbounds.entry(peer_ip).or_insert(SystemTime::UNIX_EPOCH);
+                    // Sanitize the port from the peer, if it is a remote IP address.
+                    let (peer_ip, peer_port) = match peer_ip.ip().is_loopback() {
+                        // Loopback case - Do not sanitize, merely pass through.
+                        true => (peer_ip, peer_ip.port()),
+                        // Remote case - Sanitize, storing u16::MAX for the peer IP address, passing the port through.
```

Comment:
weird phrasing here. Not sure what the comment means.

Reply:
fixed